src/DefaultBuilder/src/BootstrapHostBuilder.cs

```diff
@@ -98,36 +93,51 @@ public IHostBuilder ConfigureServices(Action<HostBuilderContext, IServiceCollect
             return this;
         }
 
-        public void RunDefaultCallbacks(HostBuilder innerBuilder)
+        public (HostBuilderContext, ConfigurationManager) RunDefaultCallbacks(HostBuilder innerBuilder)
```

Comment:
nit: isn't the config returned here the same as HostBuilderContext.Configuration? So you can avoid returning a tuple.

Reply:
I didn't want to cast

Reply:
You could also just keep the final ConfigurationManager that's being returned here the first ctor parameter like it was before. It's a bit cleaner than returning a tuple.

Reply:
I don't like the idea that this is passed in the constructor but only used in this method. I could create the configuration here and pass it in though